test/spec.git-parser.js

```diff
@@ -9,42 +8,383 @@ describe('git-parser stash show', () => {
     const res = gitParser.parseGitStashShow(text);
     expect(res).to.be.an('array');
     expect(res.length).to.be(1);
-    expect(res[0].filename).to.be('New Text Document (2).txt');
+    expect(res[0]).to.eql({ filename: 'New Text Document (2).txt' });
   });
 });
 
-describe('git-parse diff on big change', () => {
-  describe('git-parser parseGitLog', () => {
-    it('should work with branch name with ()', () => {
-      const refs = gitParser.parseGitLog('commit AAA BBB (HEAD, (test), fw(4rw), 5), ((, ()')[0].refs;
+describe('git-parser parseDiffResult', () => {
+  it('all diff selected', () => {
+    var gitDiff = 'diff --git a/package.json b/package.json\n'
+    gitDiff += 'index f71e0064..08964575 100644\n'
+    gitDiff += '--- a/package.json\n'
+    gitDiff += '+++ b/package.json\n'
+    gitDiff += '@@ -87,9 +87,10 @@\n'
```

Comment:
can we use multiline syntax with back ticks? it can be immutable and much cleaner.

Reply:
I didn't use multiline syntax because it was messing with indentation. I have just found `dedent` https://github.com/dmnd/dedent/blob/master/dedent.js which solves the issue :+1: Now it is easier to read and edit :)